vowpalwabbit/vw.vcxproj

```diff
@@ -147,93 +147,23 @@
-    <Import Project="$(SolutionDir)\packages\boost_signals-vc140.1.63.0.0\build\native\boost_signals-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_signals-vc140.1.63.0.0\build\native\boost_signals-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_system-vc140.1.63.0.0\build\native\boost_system-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_system-vc140.1.63.0.0\build\native\boost_system-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_test_exec_monitor-vc140.1.63.0.0\build\native\boost_test_exec_monitor-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_test_exec_monitor-vc140.1.63.0.0\build\native\boost_test_exec_monitor-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_thread-vc140.1.63.0.0\build\native\boost_thread-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_thread-vc140.1.63.0.0\build\native\boost_thread-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_timer-vc140.1.63.0.0\build\native\boost_timer-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_timer-vc140.1.63.0.0\build\native\boost_timer-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_type_erasure-vc140.1.63.0.0\build\native\boost_type_erasure-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_type_erasure-vc140.1.63.0.0\build\native\boost_type_erasure-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_unit_test_framework-vc140.1.63.0.0\build\native\boost_unit_test_framework-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_unit_test_framework-vc140.1.63.0.0\build\native\boost_unit_test_framework-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_wave-vc140.1.63.0.0\build\native\boost_wave-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_wave-vc140.1.63.0.0\build\native\boost_wave-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_wserialization-vc140.1.63.0.0\build\native\boost_wserialization-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_wserialization-vc140.1.63.0.0\build\native\boost_wserialization-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_zlib-vc140.1.63.0.0\build\native\boost_zlib-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost_zlib-vc140.1.63.0.0\build\native\boost_zlib-vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost-vc140.1.63.0.0\build\native\boost-vc140.targets" Condition="Exists('$(SolutionDir)\packages\boost-vc140.1.63.0.0\build\native\boost-vc140.targets')" />
-    <!--<Import Project="$(SolutionDir)\packages\CNTK.GPU.2.0-beta15\build\native\CNTK.GPU.targets" Condition="Exists('$(SolutionDir)\packages\CNTK.GPU.2.0-beta15\build\native\CNTK.GPU.targets')" />-->

```

Comment:
Make sure to keep the `$(SolutionDir)` parametrization, otherwise rl.sln will probably break. Overall, I'd like to generate a `NugetPackageRoot` property longer term.